### PR TITLE
Adding default tox jobs for runner and awx

### PR DESCRIPTION
**Pull request by matburt**
Depends-On: https://github.com/ansible/zuul-jobs/pull/9

### CODE REVIEW

**Comment by matburt** [comment]
recheck

**Comment by Spredzy** [comment]
@matburt Working sample: https://github.com/ansible/zuul-config/pull/23 - Feel free to close my PR once you've updated yours

File: zuul.d/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- project-template:
```
Comment (pabelanger) [inline zuul.d/templates.yaml]: Project templates should go into https://github.com/ansible/zuul-jobs so we can keep them untrusted, this allows for pre-merge testing.

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -1,4 +1,16 @@
 ---
+
 - project:
     name: ansible/ansible
     default-branch: devel
+
+- project:
+    name: ansible/awx
+    default-branch: devel
+    templates:
```
Comment (pabelanger) [inline zuul.d/projects.yaml]: this is not valid, and can be removed

File: zuul.d/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    name: python
+    check:
+      jobs:
+        - tox:
```
Comment (pabelanger) [inline zuul.d/templates.yaml]: This is a good start, but we should look to create new jobs in the long run, which can parent to tox and setup specific tox_envlist variables

File: zuul.d/projects.yaml (hunk 2 above)
Comment (Spredzy) [reply zuul.d/projects.yaml]: @pabelanger why is it invalid? We have such definitions in `dci-config` and its been working fine.

Comment (pabelanger) [reply zuul.d/projects.yaml]: Ah, sorry. Lack of coffee, my brain read this as Job stanza, not project. Sorry for the noise.

**Comment by Spredzy** [comment]
@pabelanger could you re-review this PR when you have time please?

**Pull request merged**